mod/messageboard/classes/Elgg/MessageBoard/Notifications/CreateMessageboardNotificationHandler.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace Elgg\MessageBoard\Notifications;
+
+use Elgg\Notifications\NonConfigurableNotificationEventHandler;
+
+/**
+ * Send a notification when a new messageboard message is posted
+ *
+ * @since 6.3
+ */
+class CreateMessageboardNotificationHandler extends NonConfigurableNotificationEventHandler {
+	
+	/**
+	 * {@inheritdoc}
+	 */
+	public function getSubscriptions(): array {
+		$result = [];
+		
+		$user = $this->getAnnotation()?->getEntity();
+		if ($user instanceof \ElggUser) {
+			$result[$user->guid] = array_keys(array_filter($user->getNotificationSettings()));
```

Comment:
we still need something like `$user->getNotificationSettings($only_return_enabled_keys)`..